test/index.js

```diff
@@ -26,39 +26,39 @@ describe('eslint-config-uphold', () => {
     const results = await linter.lintFiles([source]);
     const rules = results[0].messages.map(violation => violation.ruleId);
 
-    Array.from(rules).should.eql([
+    Array.from(rules).should.containDeep([
```

Comment:
Maybe add a `.sort()` before the should, in order to maintain the array alphabetical.

Reply:
Turns out I didn't need this in the end - I was able to switch back to the sorted array. I wasn't using the right config during dev so this was required at that point as the ordering differed. But this can be safely reverted.